Assets/NuGet/Editor/NugetHelper.cs

```diff
@@ -208,6 +211,21 @@ private static void RunNugetProcess(string arguments, bool logOuput = true)
 
             LogVerbose("Running: {0} \nArgs: {1}", files[0], arguments);
 
+#if UNITY_EDITOR_OSX
+            var monoFile = "/Library/Frameworks/Mono.framework/Versions/Current/Commands/mono";
+            //TODO:using unity mono to pack the Nuget package.
```

Comment:
Can you drop this TODO and leave a comment that we're using mono to run 'nuget.exe' - this method is used for all nuget interfacing - not just to pack things.

Reply:
Done